contribute/en/Release-Process.md

```diff
@@ -310,7 +310,14 @@ Current state: <!-- Planning|Translations (beta)|Code freeze (rc)|Released -->
   - [ ] Update the version number in `Jamulus.pro` and add the release date to the Changelog header and commit.
   - [ ] Wait for the release build to complete.
   - [ ] Contact emlynmac for signing on macOS and upload signed binary [from his](https://github.com/emlynmac/jamulus/releases/) repo [to ours](https://github.com/jamulussoftware/jamulus/releases).
-  - [ ] Do a smoke test for Windows/Mac/Linux -- Do the binaries start/connect properly? Can earlier Jamulus versions properly connect to a server based on the new release?
+  - [ ] Do a smoke test for Windows/Mac/Linux
+    - [ ] Do the binaries start/connect properly?
+    - [ ] Can earlier Jamulus versions properly connect to a server based on the new release?
+  - [ ] Do a smoke test of the upgrade notification (do this _very quickly_)
```

Comment:
Do we have any better way to test this instead of testing in production?

Reply:
Not currently -- you can't say `Jamulus --checkforupdateserver x.y.com` or something.